src/navigator-card-body.jsx

```diff
@@ -327,9 +393,18 @@
         }
     };
 
-    const handleContextMenu = () => {
-        const sel = path ? path[path.length - 1] : undefined;
-        setSelected([{ name: sel }]);
+    const handleContextMenu = (event) => {
+        let sel = getNameForEvent(event);
+        if (sel !== null && selected.length > 1) {
+            return;
+        }
+
+        if (sel === null) {
+            sel = path ? path[path.length - 1] : undefined;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-249-20240207-125527-92469164-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)